operator/v1/types_console.go

```diff
@@ -132,6 +132,10 @@ type ConsoleCustomization struct {
 	// +kubebuilder:validation:Optional
 	// +optional
 	AddPage AddPage `json:"addPage,omitempty"`
+	// perspectives allows enabling/disabling of perspective(s).
+	// +kubebuilder:validation:Optional
+	// +optional
```

Comment:
You only need one of these two, we prefer to keep `+optional`
```suggestion
	// +optional
```